ci/aws-analytical-env/shared/meta.yml

```diff
@@ -72,10 +74,12 @@ meta:
               cp ../../../../terraform-config/terraform.tfvars .
               terraform workspace show
               terraform init
+              export TF_VAR_adg_emr_launcher_zip="{base_path = \"../emr-launcher-release\", version = \"$(cat ../emr-launcher-release/version)\"}"
               terraform plan
         inputs:
           - name: aws-analytical-env
           - name: terraform-config
+          - name: emr-launcher-release
```

Comment:
This is also going to affect cognito/management-dev and cognito/management jobs which are not configured to get `emr-launcher-release`

Reply:
Should be resolved